cypress/globals.d.ts

```diff
@@ -37,6 +37,7 @@ declare namespace Cypress {
 
     getRancherResource(prefix: 'v3' | 'v1', resourceType: string, resourceId: string, expectedStatusCode: string): Chainable;
     setRancherResource(prefix: 'v3' | 'v1', resourceType: string, resourceId: string, body: string): Chainable;
+    getUserPreferences(): Chainable;
```

Comment:
Lets get this PR merged, but in another PR can you remove this line? Bad practise, i know, but quicker.

Reply:
will do!